tendermint/src/amino_types/message.rs

```diff
@@ -3,7 +3,7 @@ use std::convert::TryInto;
 
 /// Extend the original prost::Message trait with a few helper functions in order to
 /// reduce boiler-plate code (and without modifying the prost-amino dependency).
-pub trait AminoMessage: prost_amino::Message {
+pub trait Message: prost_amino::Message {
```

Comment:
👍I'm a fan of eliminating module (and crate) names from type names. Some relevant API guidelines and issues:

- [RFC#356: No module prefixes](https://github.com/rust-lang/rfcs/blob/master/text/0356-no-module-prefixes.md)
- https://github.com/rust-lang/api-guidelines/issues/66

Importing modules rather than individual types and referring to types by their module names can also help cut down on the number of imports.